toggleQuotes.js

```diff
@@ -1,7 +1,7 @@
-function handleClick() {
-  browser.toggleQuotesApi.toggleQuotes();
-};
+browser.messageDisplayAction.onClicked.addListener((tab, info) => {
+  browser.toggleQuotesApi.toggleQuotes(tab.windowId);
+});
```

Comment:
Switching from implicit function definitions to inline arrow functions. Just because :-) Using the tab provided by the listener to get the windowId and pass that into the Experiment.